src/main/java/mate/academy/Main.java

```diff
@@ -2,22 +2,39 @@
 
 import java.time.LocalDate;
 import java.time.LocalDateTime;
+import mate.academy.exception.AuthenticationException;
+import mate.academy.exception.RegistrationException;
+import mate.academy.lib.Injector;
 import mate.academy.model.CinemaHall;
 import mate.academy.model.Movie;
 import mate.academy.model.MovieSession;
+import mate.academy.model.User;
+import mate.academy.service.AuthenticationService;
 import mate.academy.service.CinemaHallService;
 import mate.academy.service.MovieService;
 import mate.academy.service.MovieSessionService;
+import mate.academy.service.UserService;
+import mate.academy.util.HashUtil;
 
 public class Main {
-    public static void main(String[] args) {
-        MovieService movieService = null;
+    private static final Injector INJECTOR = Injector.getInstance("mate.academy");
+    private static final MovieService MOVIE_SERVICE = (MovieService) INJECTOR
+            .getInstance(MovieService.class);
```

Comment:
In my opinion, it is bad practice to set the name for Services as a constant.
fix it everywhere
```suggestion
    private final Injector injector = Injector.getInstance("mate.academy");
    private final MovieService movieService = (MovieService) injector
            .getInstance(MovieService.class);
```

Reply:
> In my opinion, it is bad practice to set the name for Services as a constant. fix it everywhere

![image](https://github.com/mate-academy/hibernate-user-service-hw/assets/128720591/f47756e0-7dcc-43c1-ada1-173705ffeabe)